unityfiles/Assets/Scripts/Game/SimpleSpawner.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+
+
+public class SimpleSpawner : MonoBehaviour
+{
+    // -- inspector
+    [SerializeField]
+    private float spawnInterval;
+
+    [SerializeField]
+    private Texture2D enemyTexture;
+
+    // -- internal
+    private float lastSpawn;
+    private GameObject enemyBlueprint;
+    // Start is called before the first frame update
+    void Start()
+    {
+        this.lastSpawn = Time.time;
+        GameObject ebp = new GameObject();
+        this.enemyBlueprint = ebp;
+        ebp.name = "Enemy";
+        ebp.tag = "Enemy";
+
+
+        SpriteRenderer spriteRender = ebp.AddComponent<SpriteRenderer>();
+        spriteRender.sprite = Sprite.Create(this.enemyTexture, new Rect(0,0,this.enemyTexture.width ,this.enemyTexture.height), Vector2.zero);
+        ebp.transform.localScale = new Vector3(0.25f,0.25f,1);
+
+        BoxCollider2D bc = ebp.AddComponent<BoxCollider2D>();
+
+        Enemy enemyScript = ebp.AddComponent<Enemy>() as Enemy;
```

Comment:
Brukes disse?

Reply:
BoxCollider2D brukes for kollisjons deteksjon
enemyScript Brukes for å gi atributter til enemys 

fint om du faktisk sjekker hva enemyScript og AddComponent<BoxCollider2D> jør siden de er ganske selvsigende hvis man sjekker